### ISSUE TITLE
Resolution of the octomap in MoveIt is different from octomap message resolution

### ISSUE THREAD

**Issue by askarkg12** [Notou/Moveit-External-Octomap-Updater#1]
As I am trying to use different resolutions, there appears to be a problem with resolutions other than 0.05. 
For example, if resolution is set to 0.1, then the octomap in MoveIt is twice as small than the real resolution.
Opposite affect can be seen too. Seems like MoveIt will assume that the resolution is 0.05 automatically. 
ATM, I do not know what is causing the issue.

**Comment by Notou** [comment]
You're right, MoveIt doesn't get the octomap resolution automatically from octomap metadata. I don't remember if it could be possible for a plugin to do that for moveit.

The way I used it was to provide the resolution parameter from the launch file, like in [this one](https://github.com/Notou/ROS_quadrotor_simulator/blob/ece6ce0eed265889cf0cdf0853bac9b61da98496/quad_3dnav/launch/sensor_manager.launch.xml#L7)

Does this make sense?
